tensorrt_llm/llmapi/llm_args.py

```diff
@@ -1563,11 +1563,20 @@ class CacheTransceiverConfig(StrictBaseModel, PybindMirror):
         "Timeout in milliseconds for KV cache transfer. Requests exceeding this timeout will be cancelled."
     )
 
+    kv_transfer_sender_future_timeout_ms: Optional[int] = Field(
```

Comment:
Given the default value is 1000, is the `Optional` necessary? How about making the typing `int` directly?